### DISCUSSION TITLE
Unable to update configuration Heatit Z-Temp2

### DISCUSSION THREAD

**Discussion by duylong** [zwave-js/zwave-js#6657] [Request Support / Investigate Issue]
### Checklist

- [X] I have read and followed the above instructions
- [X] I have checked the [troubleshooting section](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index) and my problem is **not** described there.

- [X] I have read the [changelog](https://github.com/zwave-js/node-zwave-js/blob/master/CHANGELOG.md) and my problem was **not** mentioned there or the fix **did not work**.

### Describe the issue

Whatever the value, I have a failure when updating a configuration. However the module should react with Flirs... No network problem, I'm right next to the controller...
```
2024-02-02 22:33:57.985 ERROR Z-WAVE: Unable to write 1 on 112-0-7: Fail
```

### Software versions

zwave-js-ui: 9.8.2.c09b638
zwave-js: 12.4.3

### Device information

Manufacturer ID 411
Product type 4
Product ID 516
FW: v1.2.1
SDK: v7.17.2

### Checklist

- [X] I [made sure](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index?id=rule-1-use-the-driver-log-luke) to provide a **driver log** on level **debug**.

- [ ] The log includes a re-interview of the problematic device _(if applicable)_.

- [X] The log includes the problematic interaction with the device _(if applicable)_.

- [X] I provided the node ID of the problematic device _(if applicable)_.

### Upload Logfile

[zwavejs_2024-02-02.log](https://github.com/zwave-js/node-zwave-js/files/14145837/zwavejs_2024-02-02.log)

**Comment by AlCalzone** [comment]
The device does respond - the result "Fail" means that it failed to update the configuration parameter:
```
2024-02-02T21:33:43.305Z DRIVER � [Node 141] [REQ] [SendDataBridge]
                                  � source node id:   1
                                  � transmit options: 0x25
                                  � callback id:      104
                                  +-[SupervisionCCGet]
                                    � session id:      60
                                    � request updates: true
                                    +-[ConfigurationCCSet]
                                        parameter #:      7
                                        reset to default: false
                                        value size:       2
                                        value format:     UnsignedInteger
                                        value:            1

...

2024-02-02T21:33:50.459Z DRIVER � [Node 141] [REQ] [BridgeApplicationCommand]
                                  � type:        broadcast
                                  � target node: 255
                                  � RSSI:        -77 dBm
                                  +-[SupervisionCCReport]
                                      session id:          60
                                      more updates follow: false
                                      status:              Fail
                                      duration:            0s
```

I don't see why though - value size 2 and value 1 match the manuals of both available firmware version at https://heatit.com/produkt/9437/heatit-z-temp2-white-ral-9003, so it should be accepted and executed.

Honestly, this looks like a question for the Heatit support.

**Comment by duylong** [comment]
Strange, the ping on this node is successful. No failure in statistics.

I had difficulty doing the interview, this one is complete, but for another I posed a issue here https://github.com/zwave-js/zwave-js-ui/issues/3578, I can't try with another module.

I have the impression that this affects firmware 1.2 or 1.2.1, I will try to contact support.

**Reply by AlCalzone** [reply]
Not sure what pinging has to do with this, when the device tells us that it didn't accept the parameter change.

**Comment by duylong** [comment]
Just to indicate that he is alive despite the failure. In any case, I sent a support request to Heatit, wait and see.

**Reply by AlCalzone** [reply]
It's not a communication failure. Like I said, **the device tells us** that it didn't accept the parameter change.

**Comment by duylong** [comment]
Problem still present with 9.10.3 and regarding the response of "Heat it", no comment...

**Reply by AlCalzone** [reply]
What did they say? Now I'm curious...

**Reply by duylong** [reply]
Their response "we are aware of it and I've reported it to the developer. "

The logs have not changed, as you said, the device refuses the value. I tried all the settings and only got FAILS.

**Comment by duylong** [comment]
9.12.0 resolve this issue :D
